website/screens/components/box/specs/BoxSpecsPage.tsx

```diff
@@ -0,0 +1,403 @@
+              <td>
+                <Code>noneShadowDepthShadowOffsetY</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>none</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>noneShadowDepthShadowBlur</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>none</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>noneShadowDepthShadowSpread</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>none</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>oneShadowDepthShadowOffsetX</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>0x</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>oneShadowDepthShadowOffsetY</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>3px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>oneShadowDepthShadowBlur</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>6px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>oneShadowDepthShadowSpread</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>0px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>twoShadowDepthShadowOffsetX</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>0x</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>twoShadowDepthShadowOffsetY</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>3px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>twoShadowDepthShadowBlur</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>10px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>twoShadowDepthShadowSpread</Code>
+              </td>
+              <td>Container shadow</td>
+              <td>-</td>
+              <td>0px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <HeadingLink level={4}>Width</HeadingLink>
+        <DxcText as="p">
+          In the cases of <Code>fillParent</Code> and <Code>fillContent</Code>{" "}
+          the box is going to behave as a fluid container occupying the space
+          the parent leaves for its children or adapting its size to the
+          children it the box has. Instead of using a fluid approach, you can
+          set a fixed size, choosing between
```

Comment:
We should write a space at the end of this line (`{" "}`), `small` is appearing too close to the word before.

Reply:
Done!!